pkg/cluster/builder.go

```diff
@@ -0,0 +1,35 @@
+package cluster
+
+import "github.com/reearth/reearth-backend/pkg/id"
+
+type Builder struct {
+	c *Cluster
+}
+
+func New() *Builder {
+	return &Builder{c: &Cluster{}}
+}
+
+func (b *Builder) ID(cid id.ClusterID) *Builder {
+	b.c.id = cid
+	return b
+}
+
+func (b *Builder) NewID() *Builder {
+	b.c.id = id.NewClusterID()
+	return b
+}
+
+func (b *Builder) Name(n string) *Builder {
+	b.c.name = n
+	return b
+}
+
+func (b *Builder) Property(p id.PropertyID) *Builder {
+	b.c.property = p
+	return b
+}
+
+func (b *Builder) Build() (*Cluster, error) {
+	return b.c, nil
```

Comment:
```suggestion
func (b *Builder) Build() *Cluster {
	return b.c
```

This func never returns an error